libs/core/src/components/sage-avatar/sage-avatar.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sage-avatar',
+  styleUrl: 'sage-avatar.scss',
+  shadow: true,
+})
+export class SageAvatar {
+
+  /**
+   * Determines whether the badge is visible or not
+   * @defaultValue false
+   */
+  @Prop() badge? = false;
+
+  /**
+   * The src for a custom user image.
+   * @defaultValue null
+   */
+  @Prop() image?: string;
+
+  /**
+   * Preset sizes for the avatar. If a custom size is desired,
+   * use the `--size` custom property instead.
+   * @defaultValue md
+   */
+  @Prop() size?: 
+  | 'xl'
+  | 'lg'
+  | 'md' 
+  | 'sm' 
+  | 'xs';
```

Comment:
I removed the default value here because it was overriding the `--size` custom property due to CSS specificity. If we still want a default here, I can find some way to increase the specificity to have the custom prop always override the preset values or use `!important` (I do not desire this route).

Reply:
I don't think we established how we want to tackle values that have preset and custom options, but I am more than happy to set a precedent, see how the rest of the team feels about it, and add it to the best practices documentation.

Reply:
If you removed it, then you proably want to remove it from the tsdocs. It currently shows `@defaultValue md`, but we don't have a defaultValue.